test/test-applications/integrations/TestApplication.Smoke/Program.cs

```diff
@@ -35,10 +35,17 @@ public static void Main(string[] args)
             EmitTraces();
             EmitMetrics();
 
+            // The "LONG_RUNNING" environment variable is used by tests that access/receive
+            // data that takes time to be produced.
             var longRunning = Environment.GetEnvironmentVariable("LONG_RUNNING");
             while (longRunning == "true")
```

Comment:
Should we change while to if?

Reply:
Sorry missed your comment @rajkumar-rangaraj, yeah, it makes more sense to have an `if` I will change opportunistically in another PR.